pkg/secrethub/client.go

```diff
@@ -8,6 +8,10 @@ import (
 	"github.com/secrethub/secrethub-go/pkg/secrethub/internals/http"
 )
 
+const (
+	userAgentPrefix = "SecretHub/v1 GoClient/" + ClientVersion
```

Comment:
Why do we say `GoClient/vX.Y.Z` here, but do we say `secrethub-go` elsewhere as `DefaultUserAgent = "secrethub-go"`?